src/test/java/guitests/DialogMessageTests.java

```diff
@@ -40,6 +40,25 @@ public void showYesNoWarningDialogTest() throws ExecutionException, InterruptedE
     }
 
     @Test
+    public void showYesNoConfirmationDialogTest() throws ExecutionException, InterruptedException {
```

Comment:
Change this test name to follow the convention. `showYesNoConfirmationDialog_tryYesAndNo_getCorrectValue` or something similar.